src/controllers/userController.js

```diff
@@ -33,7 +33,8 @@ class UserController {
       });
       const payload = {
         id: userInfo.id,
-        username: userInfo.username
+        username: userInfo.username,
+        role: userInfo.role
```

Comment:
Why don't you destructure the needed values from userInfo here?

Reply:
Hi @okoroemeka, I agree with what @omokaro-faith said, Won't it be better if you de-structured the values, or what do you think?

Reply:
Resolved.